provider/aws/resource.go

```diff
@@ -98,11 +98,12 @@ func (p *Provider) ResourceList(app string) (structs.Resources, error) {
 	return rs, nil
 }
 
-func (p *Provider) resourceDefaults(app, resource string) (map[string]string, error) {
+func (p *Provider) ResourceDefaults(app, resource string) (map[string]string, error) {
```

Comment:
This isn't great but otherwise, I'd need to mock a lot of services and  API calls to test https://github.com/convox/rack/blob/master/provider/aws/releases.go#L171